src/components/atoms/Navbar/styles.tsx

```diff
@@ -1,76 +1,102 @@
 import styled from "styled-components";
 
-import { minDevice } from "../../../assets/styles/breakpoints";
-import { activeLink } from "../../../assets/styles/colors";
 import Wrapper from "../Wrapper";
 
-const NavbarDiv = styled(Wrapper)`
-  height: 8rem;
+import { minDevice } from "../../../assets/styles/breakpoints";
+import { activeLink, white } from "../../../assets/styles/colors";
+
+const HeaderWrapper = styled.div`
   justify-content: space-between;
   align-items: center;
+  position: relative;
+  z-index: 11;
+  display: flex;
+`;
+
+const NavbarWrapper = styled(Wrapper)`
+  height: 8rem;
+  width: 100%;
+  position: relative;
+  z-index: 12;
+  background: ${white};
+  padding-top: 3.2rem;
+  padding-bottom: 1.6rem;
+  box-sizing: border-box;
 
   &.active {
     box-shadow: 0px 1px 10px rgba(0, 0, 0, 0.08);
   }
 
   @media ${minDevice.desktopTablet} {
     height: 6.4rem;
+    padding-top: 1.6rem;
+    padding-bottom: 1.6rem;
     box-shadow: none;
   }
 
   @media ${minDevice.desktopL} {
     height: 7.2rem;
+    padding-top: 1.2rem;
+    padding-bottom: 1.2rem;
   }
 `;
 const NavbarMenu = styled.nav`
   transform: translateY(-150%);
   position: absolute;
   z-index: 2;
-
-  @media ${minDevice.desktopTablet} {
-    transform: translateY(-0%);
-    position: relative;
-  }
-`;
-
-const NavbarMenuList = styled.ol`
-  display: flex;
-  flex-direction: column;
-  justify-content: space-evenly;
   height: 11.2rem;
   transform: translateY(-150%);
   transition: transform 0.5s;
-  background-color: white;
-  position: absolute;
+  background-color: ${white};
+  display: flex;
 
   &.active {
-    transform: translateY(-30%);
+    transform: translateY(0%);
     top: 8rem;
     width: 100vw;
-    position: absolute;
     z-index: 3;
   }
 
+  @media ${minDevice.desktopTablet} {
+    transform: translateY(0);
+    position: relative;
+    padding-right: 2.4rem;
+    height: auto;
+  }
+
+  @media ${minDevice.desktopL} {
+    padding-right: 10rem;
+  }
+
+  @media ${minDevice.desktopXL} {
+    padding-right: 20rem;
+  }
+`;
+
+const NavbarMenuList = styled.ul`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  align-items: flex-start;
+  padding-left: 2rem;
+
```

Comment:
We could add it here width: 100vw to prevent strange behavior while it closing

Reply:
Good suggestion, thanks! :)

Reply:
But only for mobile devices with hamburger :D

Reply:
🍔

Reply:
Yes, I've noticed that 👩‍🎓